nixos/modules/services/misc/nzbget.nix

```diff
@@ -41,6 +42,12 @@ in {
         default = "nzbget";
         description = "Group under which NZBGet runs";
       };
+
+      configFile = mkOption {
```

Comment:
this can probably be dropped too, and just be hardcoded to `/var/lib/nzbget/nzbget.conf`.